blazor/dropdown-list/value-binding.md

```diff
@@ -150,7 +152,7 @@ First, you need to create a `DropDownList.razor` file in the `/Pages` folder as
 
 ### Usage of generic component with different type
 
-Then you can render the Generic DropDownList component with required TValue and TItem in the respective razor components. 
+Then you can render the Generic DropDownList component with required `TValue` and `TItem` in the respective razor components. 

```

Comment:
Then, render the Generic DropDownList component with the required `TValue` and `TItem` in the respective razor components.